nodejs/eks/examples/scoped-kubeconfigs/index.ts

```diff
@@ -63,10 +63,7 @@ const devsGroupRoleBinding = new k8s.rbac.v1.RoleBinding("pulumi-devs",
         subjects: [{
             kind: "Group",
             name: devsGroupName,
-        }],
-        roleRef: {
-            apiGroup: "rbac.authorization.k8s.io",
-            kind: "Role",
+        }], roleRef: { apiGroup: "rbac.authorization.k8s.io", kind: "Role",
```

Comment:
Any reason for this reformat?

Reply:
Oops, must've joined the lines by mistake during dev. Will fix this back up 👍 Thanks for catching it